modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,124 @@
++
+Conversely, if the client credentials that you provide do not match, the client application is created with no association to existing clients.
+
+
+[[migrate_with_internal]]
+== Migrate Client Applications for APIs with Anypoint Platform Client Providers
+
+To migrate your client application:
+
+. Open a text editor and paste the following commands:
++
+[source,text,linenums]
+----
+curl --location --request POST 'https://anypoint.mulesoft.com/exchange/api/v1/organizations/:orgId/applications' \
+--header 'Authorization: Bearer <omitted>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "clientId": <myClientId>,
+    "clientSecret": <myClientSecret>,
+    "grantTypes": [],
+    "name": <myApplicationName>,
+    "redirectUri": []
+}'
+----
+. Replace the `orgId` parameter with the ID of your master organization and the content of the body with the appropriate application <<properties,properties>>. 
+. Run this command at the Command prompt.
+
+[[migrate_with_external]]
+== Migrate Client Applications for APIs with External Client Providers
+
+When creating an application for an organization with registered external client providers, you can directly associate the application with the API that you want to consume, using that API instance ID (available from the API Manager UI).
+
+If you use a PingFederate external client provider or the Anypoint Platform client provider, you can include the client ID and client secret when referencing an API. Otherwise, for providers like Okta or Keycloak, which are registered using OpenID Dynamic Client registration, you cannot define client ID and client secret credentials.
+
+To migrate your client application:
+
+. Open a text editor and paste the following commands:
++
+[source,text,linenums]
+----
+curl --location --request POST 'https://anypoint.mulesoft.com/exchange/api/v1/organizations/:orgId/applications?apiInstanceId=:myApiInstanceId' \
+--header 'Authorization: Bearer <omitted>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "clientId": <myIdpClientID>,
+    "clientSecret": <myIdpClientSecret>,
+    "grantTypes": [
+        <grant type item, for example: client_credentials>
+    ],
+    "name": <myApplicationName>,
+    "redirectUri": [
+        "https://anypoint.mulesoft.com/apiplatform/:orgId/authentication/oauth2.html",
+        "https://anypoint.mulesoft.com/apiplatform/:orgId/admin/authentication/oauth2.html",
+        "https://api-notebook.anypoint.mulesoft.com/authenticate/oauth.html"
+    ]
+}'
+----
+. In the command snippet, replace the `orgId` parameter with your organization Id, `apiInstanceId` with the API instance ID, and the content of the body with the appropriate application <<properties,properties>>. 
+. Run this command at the Command prompt.
+
+[[properties]]
+== Migrate Command Properties Reference
+
+In the properties of the migrate command, provide values based on the following recommendations:
+
+* `clientId`- <Your client ID>
```

Comment:
NEVER use space-hyphen-space as a separator between a key and value; use a colon.
```suggestion
* `clientId`: <Your client ID>
```
ditto below

Reply:
Done! Thanks!